components/elm/src/main/elm_varpar.F90

```diff
@@ -48,20 +48,23 @@ module elm_varpar
   integer, parameter :: ndst        =   4     ! number of dust size classes (BGC only)
   integer, parameter :: dst_src_nbr =   3     ! number of size distns in src soil (BGC only)
   integer, parameter :: sz_nbr      = 200     ! number of sub-grid bins in large bin of dust size distribution (BGC only)
-  integer, parameter :: mxpft       =  50     ! maximum number of PFT's for any mode;
+  !integer, parameter :: mxpft       =  50     ! maximum number of PFT's for any mode;
+  integer :: mxpft                  =  50     ! maximum number of PFT's for any mode;
```

Comment:
shouldn't maxpft be a parameter constant?

Reply:
mxpft is now updated by reading the pft physiology file.

Reply:
Do all the physiology files need to be updated to read mxpft?